server/src/routes/project.ts

```diff
@@ -0,0 +1,67 @@
+import { PrismaClient } from "@prisma/client";
+import express from "express";
+
+export const projectRoutes = express.Router();
+const prisma = new PrismaClient();
+
+projectRoutes.route("/").get(async (req, res) => {
+  try {
+    // todo: find better way of retrieving numeric fields
+    ["id", "expo", "round", "table"].forEach(key => {
+      if (req.query.hasOwnProperty(key))
+        // @ts-ignore
+        req.query[key] = parseInt(req.query[key]);
```

Comment:
We want to do our best to avoid using `@ts-ignore`. So for this, you could just do, 

```
const filters = req.query.filter(key => ["expo", "round", "table"].includes(key));
const projects = await prisma.project.findMany({ where: filters});
```

I think that works?

Also, we don't want `id` to be one of the query params, as we have a different route for that.

Reply:
Ended up keeping a majority of the code seen above in my most recent push. I resolved the `@ts-ignore` issue and left some additional comments regarding why the snippet is necessary for Prisma to properly query our database. Let me know if there's anything else you need for me to do before merging. 👍🏼